scripts/build.sh

```diff
@@ -108,16 +107,6 @@ config_meson_coverage() {
         "${BUILDDIR}"
 }
 
-config_meson_appimage() {
-    CC="${CC}" "${MESON}" setup                 \
-        --werror                                \
-        --buildtype="${BUILDTYPE}"              \
-        --force-fallback-for=libnvme            \
-        --prefix=/usr                           \
```

Comment:
We should to add this prefix to the static build I think

Reply:
Come to think to it maybe we should use the build.sh static build for this as well and not the Makefile one.